sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -11,9 +11,9 @@ SELECT
 FROM
   `moz-fx-data-shared-prod.telemetry.experiment_enrollment_aggregates_hourly`
 WHERE
-  DATE(timestamp) > (
+  timestamp > (
     SELECT
-      DATE(MAX(window_end))
+      MAX(window_end)
     FROM
       `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_v1`
```

Comment:
The max `window_end` in `experiment_enrollment_aggregates_v1` for a submission_date = '2021-01-27' is `2021-01-28 00:00:00`. Comparing dates here instead of timestamps results in some missing data in the `live` view.